src/scripts/test_python.py

```diff
@@ -179,6 +179,27 @@ def test_rng(self):
 
         user_rng.add_entropy('seed material...')
 
+        if platform.system() != "Linux":
```

Comment:
Don’t do this. This breaks anyone (such as distributions or end users) which want to run the Python tests but are not using ESDM. Instead try to call ESDM. If it succeeds, great. If not, verify that we throw something indicating not implemented / not available.